openlibrary/tests/core/test_vendors.py

```diff
@@ -41,16 +40,34 @@ def test_clean_amazon_metadata_for_load_ISBN():
     assert result.get('offer_summary') is None
 
 
+amazon_titles = [
```

Comment:
Yep; note even if you're not sure if it's the "correct" thing, having a test to document what it's doing is useful, because other code might rely on this function to behave in a certain way.

Reply:
Thanks for the reviews @cdrini and @tfmorris , I have dropped the title splitting regex in favour of @cdrini's suggestion of using split, and crafted a regex to strip parentheses where they occur that can handle one level of nesting (like in that example) cleanly.  It's a bit complex, but the tests cover what it can do.